src/bosh-google-cpi/google/instance_service/google_instance_service_create.go

```diff
@@ -84,7 +84,8 @@ func (i GoogleInstanceService) Create(vmProps *Properties, networks Networks, re
 		Labels:            vmProps.Labels,
 		GuestAccelerators: acceleratorParams,
 		// Specify a non-Sandy Bridge CPU for known zones defined in the minCpuPlatform map
-		MinCpuPlatform: minCpuPlatform[vmProps.Zone],
```

Comment:
This is the only place where minCpuPlatform is used. If there is no longer a scenario where a default CPU platform causes issues, I suggest removing minCpuPlatform entirely in this file and adjust the tests accordingly.

Reply:
Removed the unnecessary code and comments. 
As far as I can see, only these two tests are relevant to the changes (both are passing, meaning we do not get Sandy Bridge CPU)
https://github.com/cloudfoundry/bosh-google-cpi-release/blob/076c3f8ee5e0250a8b7a8cb32e637ef6dd3f20b9/src/bosh-google-cpi/integration/vm_test.go#L288
https://github.com/cloudfoundry/bosh-google-cpi-release/blob/076c3f8ee5e0250a8b7a8cb32e637ef6dd3f20b9/src/bosh-google-cpi/integration/vm_test.go#L327

Both are worth keeping the way they are I think.

Reply:
Let's at least at a comment to the test why those tests exist or extend the description, indicating that in the past Sandy was the minimum provided platform and the agent is incompatible.

Reply:
Added comment and extended the tests description.